localisation/00_zuwc_united_localisation.csv

```diff
@@ -1860,7 +1860,7 @@ EVTDESC_WCDAC_150_demon;You received news that your [GetFromRelation] [From.GetF
 EVTDESC_WCDAC_150_evil;You received news that your [GetFromRelation] [From.GetFirstName] fell under the influence of dark forces, which strive to plunge Azeroth into the abyss of chaos and suffering;;;;;;;;;;;;;x
 EVTOPTB_WCDAC_150_close;[From.GetSheHeCap] is not my [GetFromRelation] anymore!;;;;;;;;;;;;;x
 escape_from_tomb_effect_tooltip;Escape from the dungeon\n;;;;;;;;;;;;;x
-is_magic_class_trigger_tooltip;Has a magic class\n;A une classe magique\n;;;;;;;;;;;;x
```

Comment:
why do you have non-artifacts related changes in the artifacts branch?

Reply:
this is related. when you got artifacts requiring magical class, this line appears. I haven't seen it earlier doing localisation translation, but while testing artifacts I payed attention...

Reply:
> this is related

`monastic_orders_break_celibate_desc` is not
`EVTDESC_WCCLS_500` is not
`EVTOPTC_WCDAC_4014` is not
`cult_of_the_damned_rank_1_male` is not
`cult_of_the_damned_rank_1_female` is not
`shadow_council_rank_1_male` is not
`shadow_council_rank_1_female` is not

Reply:
Oh, I see. I thought you meant line 1863 only, since that appeared on the front page. The rest apparently made it from the localisation branch... Some mistake while trying to switch branches with tortoise for the first time...

Reply:
Moved!